README.md

```diff
@@ -1,8 +1,11 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Unified Push Admin UI
 
-## Available Scripts
+This is the admin interface for the AeroGear Unified Push Server. It is run alongside Unified Push. 
 
-In the project directory, you can run:
+## Local Development
+
+### `npm install`
```

Comment:
We say `npm install` here, but `yarn start` after few lines. I believe we should stuck with just one.
Since `npm` doesn't require any additional installations, I would go for `npm` and replace `yarn start` with `npm run start`